<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="word_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Enter the word</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="translation1_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Enter the main translation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="translation2_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Enter the second translation</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>160</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="is_subject">
       <property name="text">
        <string>subject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="is_adjective">
       <property name="text">
        <string>adjective</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="is_verb">
       <property name="text">
        <string>verb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="is_abbreviation">
       <property name="text">
        <string>abbreviation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="sentence_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string> Enter a sentense with this word (may be with translation in brackets)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="image_link_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Enter the image link</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_word">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANGE THIS WORD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../.designer/backup/img/plus_in_circle.png</normaloff>../../.designer/backup/img/plus_in_circle.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
